localstack/services/awslambda/lambda_api.py

```diff
@@ -639,6 +639,28 @@ def get_function_code(function):
             headers={'Content-Disposition': 'attachment; filename=lambda_archive.zip'})
 
 
+@app.route('%s/functions/<function>/configuration' % PATH_ROOT, methods=['GET'])
+def get_function_configuration(function):
+    """ Get the configuration of an existing function
+        ---
+        operationId: 'getFunctionConfiguration'
+        parameters:
+    """
+    arn = func_arn(function)
+    lambda_function = lambda_arn_to_function.get(arn)
+    if not lambda_function:
+        return error_response('Function not found: %s' % arn, 404, error_type='ResourceNotFoundException')
+    result = {
+        'Version': '$LATEST',
+        'FunctionName': function,
+        'FunctionArn': arn,
+        'Handler': lambda_arn_to_handler.get(arn),
```

Comment:
There has been a change recently which aggregates the entire state of the lambdas into a single dict `arn_to_lambda`.